financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnections.kt

```diff
@@ -31,7 +31,7 @@ object FinancialConnections {
     internal fun emitEvent(
         name: Name,
         metadata: Metadata = Metadata()
-    ) {
+    ) = runCatching {
```

Comment:
this would change public API and broke older releases, is this expected?

Reply:
It won't! 

- This PR is merging into an integration branch, nothing is live yet
- Even with the above we're not changing the API vs what's already in the branch or in production.
  - emitEvent is internal, we just don't want it to crash if something goes wrong emitting.